flycheck.el

```diff
@@ -4125,6 +4137,18 @@ POS defaults to `point'."
       ;; Re-highlight the errors
       (flycheck-error-list-highlight-errors 'preserve-pos))))
 
+(defun flycheck-error-list-explain-error (&optional pos)
+  "Explain the error at POS in the error list.
+
+POS defaults to `point'."
+  (interactive)
+  (-when-let*
+      ((error (tabulated-list-get-id pos))
```

Comment:
Move this to the line above.  This kind of wrapping is really unusual for `let`-style bindings.